packages/document/main-doc/docs/zh/configure/app/experiments/source-build.mdx

```diff
@@ -0,0 +1,13 @@
+---
+sidebar_label: sourceBuild
+---
+
+# experiments.sourceBuild
+
+:::tip
+该配置由 Modern.js Builder 提供，更多信息可参考 [【源码构建模式】](https://modernjs.dev/builder/guide/advanced/source-build.html) 和 [experiments.sourceBuild](https://modernjs.dev/builder/api/config-experiments.html#experimentssourcebuild)。
```

Comment:
This file is auto generated by `packages/document/main-doc/scripts/sync.ts`, we can not manually write contents here.